### DISCUSSION TITLE
Capacity definition

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3135] [Q&A]
I am trying to analyse the capacity degradation in my model for 1000 cycles. So there is a summary variable named "Capacity [A.h]" and there is another model variable named " Discharge capacity " . But when I am using summary variable named "Capacity" it is not degrading as expected. So is there some difference between the above two variables and which should I consider to analyse capacity degradation in my cell?

**Comment by brosaplanella** [comment]
Can you please provide some more details? (e.g. a code example, a plot to illustrate what do you mean that it is not degrading as expected...)

**Reply by kumaryash7** [reply]
I am using "SEI":"solvent diffusion limited model for my LFP cell for 100 cycles only to validate the model.
The SEI parameters are from Chen2020 parameter set and my discharge capacity [A.h] vs time [h] curve is degrading too fast where as my summary variable "Capacity" Vs cycle number is not degrading significantly.
The curves looks like this:-

<img width="329" alt="Screenshot 2023-07-10 212212" src="https://github.com/pybamm-team/PyBaMM/assets/95225799/050cdcb0-94a0-42b6-ab7b-b7d82184c6ed">
<img width="251" alt="Screenshot 2023-07-10 212317" src="https://github.com/pybamm-team/PyBaMM/assets/95225799/1c2e87cd-7b84-4db3-9bd6-9ee18583a647">


Are both the capacities same or they are defined seperately?

**Reply by kumaryash7** [reply]
And even both the capacities start from a different starting capacity

**Reply by brosaplanella** [reply]
The capacity in the summary variables is the theoretical capacity (the one you would obtain at an infinitely low current), while the discharge capacity on the left is the integral of the current.
